src/layouts/error.vue

```diff
@@ -1,305 +1,85 @@
 <template>
-  <Transition name="error-dialog-" appear="appear">
-    <DialogLayout class="error-dialog">
-      <template
-        v-if="isLoginErrorFromCivicLikerRegistration"
-        #header-content
+  <div
+    class="
+      flex
+      flex-col
+      laptop:flex-row
+      items-center
+      justify-center
+      w-full
+      h-screen
+      px-[24px]
+      gap-[28px]
+      bg-shade-gray
+    "
+  >
+    <NFTWidget
+      class="shadow-lg w-full max-w-[400px]"
+      :title="$t('error_page_not_found_title_widget')"
+      :description="$t('error_page_not_found_description_widget')"
+      :price="404"
```

Comment:
should return proper error code from `error.statusCode`?

Ref: https://nuxtjs.org/docs/directory-structure/layouts#error-page